### DISCUSSION TITLE
Chrome Plugin Progress Not Updating

### DISCUSSION THREAD

**Discussion by zafarnamah** [aclap-dev/video-downloadhelper#1124] [Help and Q&A]
### Your question

I have a premium license and downloading several videos from a [LMS](https://vancouverinstitute.net/course-resources) using Chrome browser.  The videos are not downloading and also showing their true download speed. The view displayed seems to be frozen.  I tried refreshing the browser window but it had no impact on the progress bar display. 
<img width="347" alt="DownloadHelper" src="https://github.com/user-attachments/assets/4b1e634a-5871-484b-89c6-846a1ef1c416">

### Addon & CoApp version and details

Version 9.1.0.43 (stable), target google, locale en-US.
Companion App installed (v2.0.19) Recheck
Chrome Version 127.0.6533.100 (Official Build) (arm64)
MacOS Version 14.5 (23F79)

**Comment by paulrouget** [comment]
Hmm, that's new.

Mind doing this: https://github.com/aclap-dev/video-downloadhelper/wiki/How-to-enable-logs

**Reply by zafarnamah** [reply]
Please see attached.
[vdh-logs.txt](https://github.com/user-attachments/files/16576593/vdh-logs.txt)
